app/controllers/ht_registrations_controller.rb

```diff
@@ -39,6 +39,9 @@ def create
 
   def show
     fetch_presenter
+    if @registration.ip_address.present?
+      @whois = Whois::Client.new.lookup(@registration.ip_address)
```

Comment:
We might want to avoid doing this in testing; I would recommend in `config/intializers/services.rb`:

```ruby
Services.register(:whois) { Whois::Client.new }
```
and here
```ruby
@whois = Services.whois.lookup(@registration.ip_address)
```
and in `test_helper.rb` (just at the top level, I think; there's no equivalent in minitest that I could find of rspec's `before(:all)`):

```ruby
Services.register(whois) do
  Class.new do
    def lookup(ip)
      "TOTALLY LEGIT WHOIS for #{ip}"
    end
  end.new
end
```
(This creates an instance of a new anonymous class that serves as a test double for the real whois service)

Alternatively, you could look into minitest's mocking/stubbing support in setup for registrations controller tests, but I think I prefer this more explicit dependency injection approach.